tests/test_service.py

```diff
@@ -84,7 +86,30 @@ def __iter__(self):
     rp_service.collect_tests(mocked_session)
     rp_service.start_pytest_item(mocked_item)
 
-    expect(mocked_item_start.call_count == 1, 'One HTTP POST sent')
+    expect(mocked_item_start.call_count == 2, 'Two HTTP POST sent')
     code_ref = mocked_item_start.call_args[1]['code_ref']
     expect(code_ref == '/path/to/test - test_item')
     assert_expectations()
+
+
+@pytest.mark.parametrize('os_type', ('windows', 'unix'))
+def test_adding_item_to_hierarchy(mocked_item, os_type):
+    """ Test adding path of module to hierarchy on both OS
+
+    :param mocked_item:       a mocked test item
+    :param os_type:           type of operation system
+    """
+    parent_path = mocked_item.parent.fspath
+    if os_type == 'windows':
```

Comment:
Why did not u use paths directly in parametrization instated of checking for os_type?

Reply:
fixed